eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/mutable/MultiReaderHashBag.java

```diff
@@ -1201,6 +1202,14 @@ public String toStringOfItemToCount()
         {
             return (MutableBag<T>) this.delegate;
         }
+
+        @Override
+        public MutableSet<T> selectUnique()
+        {
+            MutableBag<T> selected = this.selectByOccurrences(occurrences -> occurrences == 1);
```

Comment:
This can be delegated to underlying Bag by calling 
```
        this.acquireReadLock();
        try
        {
            return this.delegate.selectUnique();
        }
        finally
        {
            this.unlockReadLock();
        }
```